doc/releases/changelog-dev.md

```diff
@@ -44,6 +44,9 @@
 
 <h4>Resource-efficient Decompositions 🔎</h4>
 
+* `qml.PCPhase` now is decomposed significantly more efficiently for more than 2 qubits.
```

Comment:
```suggestion
* The decomposition of `qml.PCPhase` is now significantly more efficient for more than 2 qubits.
```

optional, II'm not sure about the grammatical correctness of "significantly more efficiently", but it definitely bugged me 😄